content/github/getting-started-with-github/github-cli.md

```diff
@@ -13,12 +13,15 @@ topics:
 
 {% data variables.product.prodname_cli %} is an open source tool for using {% data variables.product.product_name %} from your computer's command line. When you're working from the command line, you can use the {% data variables.product.prodname_cli %} to save time and avoid switching context.
 
-You can use the following {% data variables.product.prodname_dotcom %} features with the {% data variables.product.prodname_cli %}.
+The {% data variables.product.prodname_cli %} includes {% data variables.product.prodname_dotcom %} features such as:
 
 - View, create, clone, and fork repositories
 - Create, close, and list issues and pull requests
 - Review, diff, and merge pull requests
-- Create, edit, list, and view gists
+- Create, edit, list, view, and delete gists
+- Create, list, view, and delete releases
+- Run, list, view, and list workflows
+- List, view, watch, and rerun workflow runs
```

Comment:
I'm reordering these to go from more frequently used to less frequently used features. Also, IMO, "run... workflows" already covers the final bullet point. Let's condense them into one to reduce the cognitive effort required to read the article.

```suggestion
- Run, view, and list workflows
- Create, list, view, and delete releases
- Create, edit, list, view, and delete gists
```